hack/deploy-hub-local/sno-metal3.yml

```diff
@@ -0,0 +1,23 @@
+version: stable
+tag: 4.10
+sno: true
+sno_virtual: true
+network_type: OVNKubernetes
+cluster: test-ci
+domain: alklabs.com
+memory: 25000
+numcpus: 16
+disk_size: 100
+network: bare-net
+metal3: true
+disconnected_operators:
```

Comment:
disconnected_operators only makes sense when you set disconnected to true, which will trigger creation of an additional vm to be used as disconnected registry